### PR TITLE
ERC20 utils

### PR DESCRIPTION
**Pull request by scammi**

### CODE REVIEW

File: src/charged/index.ts [hunk 1]
```diff
@@ -94,6 +95,10 @@ export default class Charged {
     return new NftService(this.state, contractAddress, tokenId);
   }
 
+  public erc20(contractAddress: string) {
```
Comment (kamwithak) [inline src/charged/index.ts:98]: a new service for getting balances and allowances exclusively for IONX? 

what is the purpose behind this

neat

File: src/charged/services/Erc20Service.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+/** 
+* @name ERC20
+* @class ERC20
+*
+* Returns a wrapped erc20 class.
+* @param {string} contractAddress
+* @return {Erc20Service}  Instance of a erc20 token standard
+* 
+*/
+export default class Erc20Service extends BaseService {
```
Comment (kamwithak) [inline src/charged/services/Erc20Service.ts:16]: extensibility of services, inheriting from base service, is fantastic

nice design decision paying dividends

Comment (scammi) [reply src/charged/services/Erc20Service.ts:16]: : )

File: src/charged/index.ts (hunk 1 above)
Comment (scammi) [reply src/charged/index.ts:98]: It just to get an erc20 abi.

**Pull request merged**